.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,94 @@
+name: Pipeline deployment
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize]
+
+env:
+  ML_IMAGE_TAG: activelearning_ml:test
+  API_IMAGE_TAG: activelearning_api:test
+  ML_ENDPOINT: train_and_predict/
+  API: api/
+  WEBAPP: webapp/
+
+jobs:
+  linting:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8' 
```

Comment:
Just curious, is there a reason for 3.10? What's the current python version used in deployment? 

I used 3.8 as it was used as the base image in the API dockerfile, and I used it to set up my local environment as well, so I chose it to prevent any breaking of dependencies and whatnot.

Reply:
No immediate reason, it's just nice to stay on the latest release. Python 3.8 is in the "security fixes only" stage now.

As far as I understand 3.8 is being used so it's good that we will use it in CI. My comment was about the future work outside of this PR.